react/interfaces.tsx

```diff
@@ -0,0 +1,9 @@
+import { VideoHTMLAttributes } from 'react'
+
+export interface VideoPlayer extends VideoHTMLAttributes<HTMLVideoElement> {
```

Comment:
Any reason to keep in a separate file?

Reply:
It's because we have 2 components using this interface

Reply:
Would be better repeat the code?

Reply:
@klzns